uavcan/equipment/ahrs/1002.RawIMU.uavcan

```diff
@@ -0,0 +1,21 @@
+#
+# Raw IMU data with timestamps
+#
+
+#
+# rad/sec
+#
+uavcan.Timestamp gyro_timestamp
+float16[3] gyro_point
+float16[3] gyro_integral
+float16 gyro_integral_dt
+float16[<=9] gyro_covariance
```

Comment:
Missing alignment here - the field `gyro_covariance` should be prepended with `void4` in order to maintain byte alignment for all elements of the array.